common/gpu/amd/default.nix

```diff
@@ -38,9 +38,9 @@
     })
     (lib.mkIf config.hardware.amdgpu.opencl {
       hardware.opengl.extraPackages = with pkgs; [
-        rocm-opencl-icd
-        rocm-opencl-runtime
+        rocmPackages.clr
+        rocmPackages.clr.icd
```

Comment:
``rocmPackages.clr`` already includes ``rocmPackages.clr.icd``.
For that matter, ``rocmPackages.clr.icd`` pulls in ``rocmPackages.clr(.out)`` as a dependency.
We only need to include ``rocmPackages.clr``. The only time we need to use ``rocmPackages.clr.icd`` explicitly is for scripts like ``VARIABLE=${rocmPackages.clr.icd}/etc/whatever``, or we just want to be explicit that we only really need the ``icd`` variant.
See: [clr/default.nix#L40-L43](https://github.com/NixOS/nixpkgs/blob/95a8f74020b744e121e3e5e639f5337c8ae1f482/pkgs/development/rocm-modules/5/clr/default.nix#L40-L43) and [clr/default.nix#L117-L119](https://github.com/NixOS/nixpkgs/blob/95a8f74020b744e121e3e5e639f5337c8ae1f482/pkgs/development/rocm-modules/5/clr/default.nix#L117-L119)

Reply:
Actually never mind, I'm wrong and completely forgot my own code.
We do need to specify ``rocmPackages.clr`` because there's no symlink. My bad!